faq/spatiotemporal-metadata.rst

```diff
@@ -0,0 +1,188 @@
+=================================
+Spatiotemporal Metadata Standards
+=================================
+
+The `International Sequence Database Collaboration (INSDC) <https://www.insdc.org/>`_ has announced `further details <>`_ on
+the new standards being introduced for reporting spatiotemporal metadata as part of our aim to make sequence data more
+findable, accessible, interoperable and reusable (FAIR). The ENA and its INSDC partners are set to introduce mandatory
+spatio-temporal information for all new samples **by the end of May 2023** unless a valid exemption is declared.
+We feel this is a sensible minimal measure that will improve the utility of INSDC sequence data that all submitters
+should be able to provide and where this can not,
+`suitable reporting options <https://www.insdc.org/submitting-standards/missing-value-reporting/>`_ are available.
+We look forward to seeing the positive impact this has on incoming data.
+
+Implementation plans are shared in the full `INSDC announcement <>`_ with ENA-specific details below.
```

Comment:
URL missing for 'INSDC announcement'

Reply:
Hi @ihsann1, thank you for this! I think you are reviewing an outdated commit as I added these links this morning and it looks fine to me: 
https://github.com/enasequence/read_docs/pull/94/files

Reply:
Yes, sorry reloaded to the updated version